packages/realm-server/server.ts

```diff
@@ -47,27 +50,56 @@ export function createRealmServer(realms: Realm[]) {
         return;
       }
 
+      let maybeAssetPath = resolve(join(distPath, req.url));
+      if (
+        maybeAssetPath === distPath &&
+        realms.length > 0 &&
+        req.headers.accept?.includes('text/html')
+      ) {
+        // this would only be called when there is a single realm on this
+        // server, in which case just use the first realm
+        res.setHeader('Content-Type', 'text/html');
```

Comment:
try serving a 302 to add the slash

Reply:
actually a redirect won't work here since we need to rewrite the index.html with the realm specific config